src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -7620,6 +7620,10 @@ ExpressionSyntax node4
             Assert.Equal(MethodKind.BuiltinOperator, symbol1.MethodKind);
             Assert.True(symbol1.IsImplicitlyDeclared);
 
+            var synthesizedMethod = compilation.CreateBuiltinOperator(
```

Comment:
>CreateBuiltinOperator [](http://example.com/codeflow?start=48&length=21)

I think it makes sense to add similar validation to Microsoft.CodeAnalysis.Test.Utilities.TestOperationVisitor.VisitUnaryOperator. We have a test leg that pushes virtually every test scenario that we have in compiler's tests through that validator. So, we should be able to catch any form of unary operator for which we have a test today, or will have a test in the future.

Reply:
Note, that IOperation nodes probably don't return symbols for intrinsic operators, so they will have to be requested from semantic model based on syntax.

Reply:
done.